source/configuration_manual/authentication/lua_based_authentication.rst

```diff
@@ -56,11 +87,18 @@ Example
     return dovecot.auth.PASSDB_RESULT_OK, { ["password"]=password }
   end
 
+  function auth_passdb_get_cache_key()
+    return "lua-passdb-1-%u%d"
```

Comment:
It's not necessary to identify the passdb/userdb uniquely - that's done automatically. Also %u%d is a bit bad example because %d is included in %u. A better one could be e.g. `return "%u%{service}"`